website/docs/share-single.mdx

```diff
@@ -94,6 +94,7 @@ You can pass the option that will be handled by the native code, similar to `Sha
 | **LINKEDIN**          |   ✅    | 🚫  | 🚫      |
 | **TELEGRAM**          |   ✅    | ✅  | 🚫      |
 | **VIBER**             |   ✅    | ✅  | 🚫      |
+| **TWITTER/X**         |   🚫    | ✅  | 🚫      |
```

Comment:
seems that we have the file for android but I didn't test yet, not sure if it's working or if anyone is using